Frontend/src/GoogleMap/GoogleMap.js

```diff
@@ -160,7 +159,13 @@ const GoogleMap = () => {
             ref={markerRef}
             position={{ lat: addPlaceLocation.lat, lng: addPlaceLocation.lng }}
           >
-            <Pin background={'#FBBC04'} glyphColor={'#000'} borderColor={'#000'} />
+            <div className='w-9 h-9 bg-gray-400 rounded-t-full rounded-bl-full rotate-45 border border-black'>
+              <img
+                className='-rotate-45 w-6 h-6 absolute transform left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2'
+                src={`./assets/plus_icon.svg`}
+                alt='plus_icon'
+              />
+            </div>
```

Comment:
Można by było dać to do oddzielnego componentu ten fragment + ten niżej. Nie jest to konieczne

Reply:
Jasne już poprawiam